IntoHistory/MainView/support Views/MainViewButton.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  MainViewButton.swift
+//  IntoHistory
+//
+//  Created by Yu ahyeon on 2022/08/12.
+//
+
+import UIKit
+
+class MainViewButton: UIView {
+
+    // MARK: - Property
+
+    private let buttonShape: UIView = {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = UIColor.white
+        view.layer.cornerRadius = 10
+        return view
+    }()
+
+    let buttonTitle: UILabel = {
+        let title = UILabel()
+        title.font = UIFont.boldSystemFont(ofSize: 22)
+        return title
+    }()
+
+    let buttonSubtitle: UILabel = {
+        let subtitle = UILabel()
+        subtitle.font = UIFont.systemFont(ofSize: 16)
+        subtitle.text = "내용"
+        return subtitle
+    }()
+
+    lazy var labelStack: UIStackView = {
```

Comment:
p1;

현재 여기 labelStack도 MainViewButton 내부에서만 사용되고 있기 때문에 접근제어자가 필요해보입니다아 🤔
```suggestion
    private lazy var labelStack: UIStackView = {
```

Reply:
넵!

Reply:
반영했습니다.
17b65ba76c2de87fe9b7327b86c8bc5033cf8a08